day_two_guide/topics/proc_restoring-etcd.adoc

```diff
@@ -288,7 +288,8 @@ To restore etcd on a static pod:
 to another directory:
 +
 ----
-$ mv /etc/origin/node/pods/etcd.yaml .
+# mkdir -p /etc/origin/node/pods-stopped
```

Comment:
@vwalek You're saying that these commands need to be run as root, correct? However, the rest of the procedure still uses `$`, indicating not root. Can you update so that all steps in this procedure use `#` (indicate being root through the whole procedure) instead of switching back and forth? Thanks!

Reply:
Sure, let me check that.